README.md

```diff
@@ -115,6 +124,27 @@ file, they can be found in [`scripts`](./docker/package/scripts) and
 [`defaults`](./docker/package/defaults) folders respectively.
```

Comment:
It's not clear what to do with these files, but probably it's something to be described in `systemd` documentation rather than ours. However, if there is a link to something nice that explains what to do with these files (some cool `systemd` documentation), I think it makes sense to include it here.

Reply:
Generated service file describes the required path for the `EnvironmentFile` (in case `EnvironmentFile` is needed).

E.g. generated baker service will have something like
```
...
EnvironmentFile=/etc/default/tezos-baker-006-PsDELPH1
...
```

Reply:
What is `EnvironmentFile`? Is it the same kind of file as "default configuration file"?

Reply:
yes, it's a file where additional environment variables are defined